x-pack/plugins/security_solution/public/management/pages/policy/view/protection_updates/protection_updates_layout.tsx

```diff
@@ -358,6 +360,11 @@ export const ProtectionUpdatesLayout = React.memo<ProtectionUpdatesLayoutProps>(
 
     return (
       <>
+        <ProtectionUpdatesWarningPanel
+          showCallout={showWarningPanel}
+          onClose={() => setShowWarningPanel(false)}
```

Comment:
@caitlinbetz At this point I set it to be rendered on every visit - if user closes it and reenters the tab or refresh the page it will come up again. Should we persist its state so once closed it wont appear again?